testing/README.md

```diff
@@ -0,0 +1,219 @@
+# Implementation Report
+
+This directory contains the
+[WoT Thing Description Implementation report](https://w3c.github.io/wot-thing-description/testing/report.html)
+and the sources for it.
+The report itself is in [report.html](report.html).
+Do not edit this file,
+it is autogenerated by executing `npm run assertions` from the main directory,
+and any edits would be overwritten the next time you generate it.
+
+## Adding Your Results
+
+If you want to update your input for a Member organization, please go to [WoT IG testing](https://github.com/w3c/wot/tree/master/testing/tests/2019-05) and see the [README](https://github.com/w3c/wot/blob/master/testing/tests/2019-05/README.md) there.
+
+## Updating Implementation Report
+
+### w3c/wot Repository: Updating Results from Inputs
+
+Test inputs are collected via the WoT IG and are currently in `wot/testing/tests/2019-05`.
```

Comment:
This looks like an old link, please check.

Reply:
yeah, see above.   Whole file was obsolete.